src/sandstorm/cgroup2.h

```diff
@@ -0,0 +1,35 @@
+#ifndef SANDSTORM_CGROUP2_H_
+#define SANDSTORM_CGROUP2_H_
+
+#include <sys/types.h>  // For pid_t
+#include <kj/io.h>
+
+namespace sandstorm {
+  class Cgroup {
```

Comment:
Nitpick: We don't normally indent the body of a namespace.

(Also in the .c++ file.)